kong/plugins/aws-lambda/handler.lua

```diff
@@ -63,6 +63,27 @@ local function get_now()
 end
 
 
+local function validate_http_status_code(status_code)
+  if not status_code then
+    return false
+  end
+
+  if type(status_code) == "string" then
+    status_code = tonumber(status_code)
+  end
+
+  if not status_code then
+    return false
+  end
```

Comment:
This should be moved inside the `if` block above, if the `status_code` is not `string` then there is no reason to check it again here.

Reply:
Fixed

I wonder if it needs more strict check on types, like

```
  if type(status_code) == "string" or type(status_code) == "number" then
    status_code = tonumber(status_code)

    if not status_code then
      return false
    end

  else
    -- Validation failed if status code is not string or number
    return false
  end
  
  end
```